src/static/helpers/types.ts

```diff
@@ -59,3 +59,19 @@ export interface QueryParameters {
  * Generic interface for all parameter types.
  */
 export type UrlParameters = PathParameters | QueryParameters;
+
+/**
+ * Custom query parameter type with any string prefixed with `c_` as the key and the allowed
+ * types for query parameters for the value.
+ */
+export type CustomQueryParameters = {
+  [key in `c_${string}`]: string | number | boolean | string[] | number[];
+};
+
+/**
+ * Custom body request type with any string prefixed with `c_` as the key and the allowed
+ * types for the value.
+ */
+export type CustomRequestBody = {
+  [key in `c_${string}`]: string | number | boolean | string[] | number[];
```

Comment:
EDIT: sorry I missed this, we should update this type as we can pass in complex objects into the request body, I believe something like this should fix our use case

```suggestion
  [key in `c_${string}`]: string | number | boolean | string[] | number[] | { [key: string]: any };
```